dist/pr.tpl

```diff
@@ -21,6 +21,13 @@ Release author: @${author}
 - No major or minor tags will be updated as configured
 <% } %>
 
+#### Artifacts
+<% if (artifactAttached) { %>
+- An artifact will be attached to the release
+<% } else { %>
+- No artifacts will be attached to the release
+<% } %>
```

Comment:
since we're describing this in the PR, why not include the path of the artifact as well?